File: app/src/test/java/com/farmerbb/taskbar/mockito/StringAnswer.java
package com.farmerbb.taskbar.mockito;

import org.mockito.invocation.InvocationOnMock;
import org.mockito.stubbing.Answer;

public class StringAnswer implements Answer<String> {
    public String answer = null;

    @Override
    public String answer(InvocationOnMock invocation) throws Throwable {
        return answer;
    }
}

File: app/src/test/java/com/farmerbb/taskbar/mockito/StringAnswer.kt
package com.farmerbb.taskbar.mockito

import org.mockito.invocation.InvocationOnMock
import org.mockito.stubbing.Answer

class StringAnswer(answerValue: String) : Answer<String> {
    private var answer: String = answerValue

    @Throws(Throwable::class)
    override fun answer(invocation: InvocationOnMock): String {
        return answer
    }
}
